exporters/otlp/src/otlp_log_recordable.cc

```diff
@@ -17,8 +17,10 @@ namespace otlp
 
 proto::resource::v1::Resource OtlpLogRecordable::ProtoResource() const noexcept
 {
-  // TODO Populate shared resource
-  return private_resource_;
+  proto::resource::v1::Resource proto;
+  OtlpRecordableUtils::PopulateAttribute(
+      &proto, opentelemetry::sdk::resource::Resource::Create(resource_attributes_));
```

Comment:
This is fine for now. But the actual fix should be the improvement in LoggerProvider, to let the resources and instrumentation library passed through it ( similar to what we do with TracerProvider). I will create an issue for the improvements.

Reply:
Agree strongly. I could help to do this if there is any need

Reply:
Thanks. Have added #1086 if you would like to pick it up.